### PR TITLE
Clarify 2.6.0 upgrade instructions

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: docs-source/content/_index.md [hunk 1]
```diff
@@ -24,11 +24,12 @@ using the operator to deploy and run a WebLogic domain container-packaged web ap
...
 
 The [current release of the operator](https://github.com/oracle/weblogic-kubernetes-operator/releases) is 3.0.0.
-This release was published on July 1, 2020. See the operator prerequisites and supported environments [here]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
+This release was published on July 17, 2020. See the operator prerequisites and supported environments [here]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
 
-This release introduces _non-backward compatible_ changes; however, this release can be run in the same
-cluster as the 2.6.0 version of the operator allowing for staged migration.  You can also upgrade from 2.6.0 to 3.0.0 without needing to restart or recreate
-any existing domains.
+This release introduces _non-backward compatible_ changes; however, operators using this release can be run in the same
+Kubernetes cluster as operators using the 2.6.0 version allowing for staged migration.  You can replace an operator using 2.6.0 with an operator using a 3.x release without needing to recreate
```
Comment (rosemarymarano) [inline docs-source/content/_index.md:30]: Slightly simpler wording if you state, "You can replace a 2.6.0 operator with a 3.x operator without needing to recreate any existing domains"

File: docs-source/content/_index.md [hunk 2]
```diff
@@ -24,11 +24,12 @@ using the operator to deploy and run a WebLogic domain container-packaged web ap
...
+any existing domains; however, you must delete the 2.6.0 Helm release and then install this version rather than using a Helm upgrade. When the 3.x operator starts, it will roll any running WebLogic Server instances
```
Comment (rosemarymarano) [inline docs-source/content/_index.md:31]: and then install this version -> (here I would suggest stating the version) and then install the 3.x version

File: docs-source/content/userguide/managing-operators/installation/_index.md [hunk 3]
```diff
@@ -86,6 +86,30 @@ $ helm install weblogic-operator weblogic-operator/weblogic-operator
 
 #### Upgrade the operator
 
+{{% notice note %}}
+Because operator 3.0.0 introduces _non-backward compatible_ changes, you can not use `helm upgrade` to upgrade
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-operators/installation/_index.md:90]: can not -> cannot (one word)

File: docs-source/content/userguide/managing-operators/installation/_index.md [hunk 4]
```diff
@@ -86,6 +86,30 @@ $ helm install weblogic-operator weblogic-operator/weblogic-operator
...
+an operator using 2.6.0 to an operator using 3.0.0. Instead, you must delete the 2.6.0 operator and then install the
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-operators/installation/_index.md:91]: (Again, here I would suggest the less verbose way of describing these operator versions)  -> a 2.6.0 operator to a 3.0.0 operator

File: docs-source/content/userguide/managing-operators/installation/_index.md [hunk 5]
```diff
@@ -86,6 +86,30 @@ $ helm install weblogic-operator weblogic-operator/weblogic-operator
...
+3.0.0 operator. 
+
+The deletion of the 2.6.0 operator will _not affect_ the Domain CustomResourceDefinition (CRD) and will _not stop_ any
+WebLogic Server instances already running.
+
+When the 3.0.0 operator is installed it will automatically roll any running WebLogic Server instances created by the 2.6.0 operator.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-operators/installation/_index.md:97]: installed -> installed, (comma)

File: docs-source/content/userguide/managing-operators/installation/_index.md [hunk 6]
```diff
@@ -86,6 +86,30 @@ $ helm install weblogic-operator weblogic-operator/weblogic-operator
...
+This rolling will preserve WebLogic cluster availability guarantees (for clustered members only) similarly to any other rolling.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-operators/installation/_index.md:98]: rolling -> rolling restart

File: docs-source/content/release-notes.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
-| July 1, 2020 | v3.0.0 | yes | Adds Model in Image feature and support for Istio. Removal of support for Helm 2.x. Operator performance improvements to manage many domains in the same Kubernetes cluster. |
+| July 17, 2020 | v3.0.0 | yes | Adds Model in Image feature and support for Istio. Removal of support for Helm 2.x. Operator performance improvements to manage many domains in the same Kubernetes cluster. |
```
Comment (mriccell) [inline docs-source/content/release-notes.md:11]: It is missing these two important features/certifications

OLCNE 1.1 certification, 

For Domain Home on PV added the ability to apply topology changes to the domain which the Operator can now uptake without requiring any downtime.

Comment (rosemarymarano) [reply docs-source/content/release-notes.md:11]: We added this to the 2.6.0 release notes, "Certified support of Oracle Linux Cloud Native Environment (OLCNE) 1.1 with Kubernetes 1.17.0."

Comment (rjeberhard) [reply docs-source/content/release-notes.md:11]: Monica, I'll add a statement about applying topology changes. Is it okay that the OLCNE certification is just on the 2.6.0 release? It's okay with me if we copy it over.

Comment (mriccell) [reply docs-source/content/release-notes.md:11]: yes

**Comment by rjeberhard** [comment]
Okay, thanks @mriccell. Please me know what you think.

**Pull request merged**